<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-01-09T20:21:43 IST" name="Default suite" finished-at="2022-01-09T20:22:03 IST" duration-ms="20455">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.ebanking.testCases.FirstRunner@3590fc5b]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test started-at="2022-01-09T20:21:43 IST" name="Default test" finished-at="2022-01-09T20:22:03 IST" duration-ms="20455">
      <class name="com.ebanking.testCases.FirstRunner">
        <test-method is-config="true" signature="setUpClass()[pri:0, instance:com.ebanking.testCases.FirstRunner@3590fc5b]" started-at="2022-01-09T20:21:43 IST" name="setUpClass" finished-at="2022-01-09T20:21:43 IST" duration-ms="426" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.ebanking.testCases.FirstRunner@3590fc5b]" started-at="2022-01-09T20:21:43 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2022-01-09T20:22:03 IST" duration-ms="19903" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["verify login of ebanking home page"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[To check the login functionality of banking application]"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertFalse(Assert.java:67)
at org.testng.Assert.assertFalse(Assert.java:77)
at org.testng.asserts.Assertion$4.doAssert(Assertion.java:116)
at org.testng.asserts.Assertion.executeAssert(Assertion.java:25)
at org.testng.asserts.Assertion.doAssert(Assertion.java:12)
at org.testng.asserts.Assertion.assertFalse(Assertion.java:112)
at com.ebanking.testCases.TC_LoginPage_1.user_can_verify_the_title(TC_LoginPage_1.java:130)
at âœ½.user can verify the title(file:///D:/git/BDD_YOUTUBE_ENGLISH/Ebankproject/src/test/resources/Features/LoginPage.feature:9)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:com.ebanking.testCases.FirstRunner@3590fc5b]" started-at="2022-01-09T20:22:03 IST" name="tearDownClass" finished-at="2022-01-09T20:22:03 IST" duration-ms="23" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- com.ebanking.testCases.FirstRunner -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
